text/0113-canonical-metadata.md

```diff
@@ -0,0 +1,625 @@
+                    }
+                },
+                "staticResources": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/StaticResourceReference"
+                    }
+                },
+                "dynamicChildren": {
+                    "type": "array",
+                    "items": { 
+                        "$ref": "#/definitions/DynamicComponentReference"
+                    }
+                },
+                "directives": {
+                    "type": "object",
+                    "properties": {
+                        "forEach": {
+                            "type": "array", 
+                            "items": { "$ref" : "#/definitions/TemplateForEachDirective" }
+                        },
+                        "iterator": {
+                            "type": "array", 
+                            "items": { "$ref" : "#/definitions/TemplateIteratorDirective" }
+                        },
+                        "if": {
+                            "type": "array", 
+                            "items": { "$ref" : "#/definitions/TemplateIfDirective" }
+                        },
+                        "lwcDom": {
+                            "type": "array", 
+                            "items": { "$ref" : "#/definitions/TemplateLwcDomDirective" }
+                        }
+                    }
+                }
+            }
+        },
+        "JSFile": {
+            "type": "object",
+            "properties": {
+                "fileType": { "$ref": "#/definitions/FileType" },
+                "fileName": { "type": "string" },
+                "componentClasses": {
+                    "type": "array",
+                    "items": { 
+                        "$ref": "#/definitions/Class"
+                    }
+                },
+                "imports": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Import"
+                    }
+                },
+                "dynamicImports": {
+                    "type": "array",
+                    "items": { 
+                        "$ref": "#/definitions/DynamicImport"
+                    }
+                },
+                "exports": {
+                    "type": "array",
+                    "items": [
+                        { "$ref": "#/definitions/Export"},
+                        { "$ref": "#/definitions/AggregatingExport"}
+                    ]
+                },
+                "programmaticEvents": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ProgrammaticEvent"
+                    }
+                },
+                "staticResources": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/StaticResourceReference"
```

Comment:
What would a static resource reference be in a JavaScript module?

Reply:
A few examples:
1. Programmatically injected elements that load a static resource
2. Resources loaded using the platformResourceLoader.

There are more examples in this document from LMR(https://salesforce.quip.com/9WbbA0wTX9lE)